src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionsBase.cs

```diff
@@ -3010,7 +3060,7 @@ private bool HasExplicitArrayConversion(TypeSymbol source, TypeSymbol destinatio
                     var sourceElement = ((NamedTypeSymbol)source).TypeArgumentWithDefinitionUseSiteDiagnostics(0, ref useSiteDiagnostics).TypeSymbol;
                     var destinationElement = destinationArray.ElementType.TypeSymbol;
 
-                    if (HasIdentityConversion(sourceElement, destinationElement))
+                    if (HasIdentityConversionInternal(sourceElement, destinationElement))
```

Comment:
>HasIdentityConversionInternal [](start = 24, length = 29)

Why not drop nullability here as well?

Reply:
Is it necessary or will one of `HasImplicitReferenceConversion` or `HasExplicitReferenceConversion` return `true` in cases where the difference is limited to nullability?

---
In reply to: [168041977](https://github.com/dotnet/roslyn/pull/23981#discussion_r168041977) [](ancestors = 168041977)